USAGE

```diff
@@ -0,0 +1,54 @@
+register_flow pipeline
+=====================
+
+Register t1, metrics and tractograms to template space.
+We suggest to use https://github.com/scilus/combine_flows to create input.
+
+
+USAGE
+
+nextflow run main.nf [OPTIONAL_ARGUMENTS] --input --template
+
+DESCRIPTION
+
+    --input=/path/to/[root]     Root folder containing multiple subjects
+                                                [root]
+                                                ├── S1
+                                                │   │── *t1.nii.gz (diff space)
+                                                |   └── metrics (optional)
+                                                |   │   └── *.nii.gz
+                                                |   └── tractograms (optional)
+                                                |       └── *.trks
+                                                └── S2
+                                                    └── *
+
+    --template=/path/to/[template]              Path to a nifti T1 template used for registration.
+
+OPTIONAL ARGUMENTS (current value)
+    --linear                                    If set, it will use linear registration instead of non linear registration.
+
+    --slow_registration                         If set, it will use antsRegistrationSyN instead of antsRegistrationSyNQuick for registration.
```

Comment:
add default in ().
Also, please add the slow option as default and rename this option --quick_registration

I think we should always use slow registration unless we are prototyping

Reply:
Just tested and the --slow_registration option went super fast on Cipaye on 12 subjects. Are you sure that you are truly activating the non-quick version? :)